### DISCUSSION TITLE
Tested in Brazil with Vivo.

### DISCUSSION THREAD

**Discussion by Balooski** [kyujin-cho/pixel-volte-patch#285] [Carrier-specific Experiences]
VoLTE is working. VoWifi is not, but I believe it to be due to the carrier. Shall test 5G when I get to a coverage area. If successfull, I shall update.

**Comment by samuelgarabini** [comment]
Did you solve it? I tested here, but only VoLTE worked

**Comment by dperez83** [comment]
> VoLTE is working. VoWifi is not, but I believe it to be due to the carrier. Shall test 5G when I get to a coverage area. If successfull, I shall update.

Are you using Android 15? VoWifi worked for Vivo but after the Android 15 update the IMS registration is never completed and VoWifi does not work.

Does the IMS registration is completed for you? Do you see your phone number on Android sim card settings?

![Screenshot_20241111-092108~2](https://github.com/user-attachments/assets/aa7b2515-755f-4d40-9441-422018ddb6f8)

![Screenshot_20241111-092416~2](https://github.com/user-attachments/assets/19e8aa11-ed72-4ebd-a7d1-d1db6416f4b2)

**Reply by samuelgarabini** [reply]
In my phone, only VoLTE works and IMS is registered. My SIM Card only shows the latest numbers of my number, where do you go to see this config? If I go to the settings in "About system", I can go to the SIM card and view the entire number.

When it worked for you, did you put which IMS config in APN? I don't know if I did something wrong.

![Screenshot_20241111-122519](https://github.com/user-attachments/assets/83398b01-40d9-4628-9115-84e170f20526)

**Reply by dperez83** [reply]
> In my phone, only VoLTE works and IMS is registered. My SIM Card only shows the latest numbers of my number, where do you go to see this config? If I go to the settings in "About system", I can go to the SIM card and view the entire number.

Settings > Network and internet > SIMs

> 
> When it worked for you, did you put which IMS config in APN? I don't know if I did something wrong.
> 
I never touched the IMS APN. Are you using Pixel with Android 15?

**Reply by samuelgarabini** [reply]
I'm using the Android 14 yet, but if I update I'll tell you. The only thing I did was to uninstall an app called "Google Wifi Provisioner", but I searched about it and didn't see anything related to VoWifi... my option is try to format the data in my phone and try again.

**Reply by dperez83** [reply]
Did you update to Android 15? I am curious to know if the IMS registration stopped working for you after the update.

**Reply by samuelgarabini** [reply]
> Did you update to Android 15? I am curious to know if the IMS registration stopped working for you after the update.

No. I didn't update to Android 15
